### ISSUE TITLE
class Neural_net(nn.Module) in Defining the neural net

### ISSUE THREAD

**Issue by marmatti** [Edinburgh-Chemistry-Teaching/ATCP_23_24#8]
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/5d98f1f1-b579-432e-a8e8-a9b892f46c5d)

I don't understand: **we thus have to set the seed back to 2**
